### PR TITLE
Set Sticky navigation bar on scroll

### PR DESCRIPTION
**Pull request by Xaviju**
Link to this issue: https://tree.taiga.io/project/taiga/issue/3653

Test acceptance: 

- Go to a page with long scroll. For example, a Story detail
- Scroll down. Check that navigation bar stays in the left side sticky
- Scroll up to the top. Check that navigation bar goes back to its position.
- Click on it in different positions, try to break it.

Please review JS code.

### CODE REVIEW

File: app/modules/components/project-menu/project-menu.directive.coffee [hunk 1]
```diff
@@ -31,6 +31,15 @@ ProjectMenuDirective = (projectService, lightboxFactory) ->
             return projectService.project
         ), projectChange
 
+        scope.$watch ( () ->
+            return position = $(window).scrollTop()
+        ), (position) ->
+            console.log position
```
Comment (bameda) [inline app/modules/components/project-menu/project-menu.directive.coffee]: Hi! I am a missed console log :pig_nose:

**Comment by Xaviju** [comment]
Fixed!

**Comment by Xaviju** [comment]
@bameda Weird behavior fixed! It needed a nice $digest

**Comment by bameda** [comment]
:smiley_cat:

**Comment by superalex** [comment]
image

**Pull request merged**